.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+name: hersh-CI
+on: [push, pull_request]
```

Comment:
Will this run when a commit is pushed to an open PR?

Reply:
Yes.